host.go

```diff
@@ -206,9 +206,26 @@ func (h *Host) ConfigureAuth() error {
 		return nil
 	}
 
-	ip, err := h.Driver.GetIP()
-	if err != nil {
-		return err
+	var (
+		ip         = ""
+		ipErr      error
+		maxRetries = 4
+	)
+
+	for i := 0; i < maxRetries; i++ {
+		ip, ipErr = h.Driver.GetIP()
+		if ip != "" {
+			break
+		}
+		time.Sleep(5 * time.Second)
```

Comment:
Would it be reasonable to log a debug message in this case letting the user know that `machine` is waiting for an IP address?